srcpkgs/quazip/template

```diff
@@ -9,21 +9,16 @@ maintainer="Duncaen <duncaen@voidlinux.org>"
 license="LGPL-2.1-only"
 homepage="https://github.com/stachenov/quazip"
 distfiles="https://github.com/stachenov/quazip/archive/v${version}.tar.gz"
-checksum=4fda4d4248e08015b5090d0369ef9e68bdc4475aa12494f7c0f6d79e43270d14
+checksum=79633fd3a18e2d11a7d5c40c4c79c1786ba0c74b59ad752e8429746fe1781dd6
 
 if [ -n "$CROSS_BUILD" ]; then
 	hostmakedepends+=" qt5-host-tools qt5-devel"
 fi
 
 quazip-devel_package() {
+	build_style=meta
 	depends="${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
-	pkg_install() {
-		vmove usr/include
-		vmove usr/share
-		vmove "usr/lib/*.a"
-		vmove "usr/lib/*.so"
-	}
```

Comment:
this is incorrect, just some files were moved around and the -devel package should not be empty.

```diff
@@ -20,8 +20,8 @@ quazip-devel_package() {
        short_desc+=" - development files"
        pkg_install() {
                vmove usr/include
-               vmove usr/share
-               vmove "usr/lib/*.a"
+               vmove usr/lib/cmake
+               vmove usr/lib/pkgconfig
                vmove "usr/lib/*.so"
        }
 }
```